conn_go19_test.go

```diff
@@ -0,0 +1,71 @@
+// +build go1.9
+
+package pq
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestArrayArg(t *testing.T) {
+	db := openTestConn(t)
+	defer db.Close()
+
+	for _, tc := range []struct {
+		name string
+		want interface{}
+	}{
+		{
+			name: "array-value",
+			want: [...]int64{245, 231},
+		},
+		{
+			name: "slice-value",
+			want: []int64{245, 231},
+		},
+		{
+			name: "array-pointer",
+			want: &[...]int64{245, 231},
+		},
+		{
+			name: "slice-pointer",
+			want: &[]int64{245, 231},
+		},
+	} {
+		t.Run(tc.name, func(t *testing.T) {
+			r, err := db.Query("SELECT $1::int[]", Array(tc.want))
```

Comment:
This is calling `Array` but it looks like `CheckNamedValue` is supposed to be doing that.

Reply:
I'm confused. This is on the query side, `CheckNamedValue` is on the response side.

Reply:
Oh, nevermind - you are correct. Let me remove it.